dbcon/joblist/groupconcat.cpp

```diff
@@ -1038,17 +1064,18 @@ void GroupConcatNoOrder::getResult(uint8_t* buff, const string& sep)
       outputRow(oss, fRow);
       fRow.nextRow();
     }
-
+    size_t sizeDiff = oss.str().size() - prevResultSize;
```

Comment:
Could it be a situation when prevResultSize > oss.str().size(), if so we could face possible signed integer overflow.

Reply:
How it is possible given that oss size is a monotonic non-decreasing function?

Reply:
Good to know, never heard about it.

Reply:
I mean every iteration of a while loop either adds or doesn't change oss so sizeDiff can't be negative.